interfaces/devdoc/platform_requirements.md

```diff
@@ -12,28 +11,28 @@ implementers
 may find it makes more sense to perform these operations outside of the scope of the Azure IoT SDK.
 In that case, the `platform_init` and `platform_deinit` calls may be left empty.
 
-###   Exposed API
+### Exposed API
+
 The platform adapter must implement `platform_init` and `platform_deinit`.
 
-###   platform_init
+### platform_init
 
 The `platform_init` call performs any global initialization necessary for a particular platform.
 
 ```c
 int platform_init();
 ```
 
-**SRS_PLATFORM_30_000: [** The `platform_init` call shall perform any global initialization needed by the platform and return 0 on success. **]**
-
-**SRS_PLATFORM_30_001: [** On failure, `platform_init` shall return a non-zero value. **]**
+**NON_PLATFORM_30_000: [** The `platform_init` call shall perform any global initialization needed by the platform and return 0 on success. **]**
```

Comment:
>NON [](http://example.com/codeflow?start=2&length=3)

Why did all the "SRS get changed to "NON" in this PR?

Reply:
These SRS don't correspond to code, but are here just in general and the traceability was flagging all these as missing in code and tests.  I didn't want to remove the files just yet until all the linux specs get written.